src/PollController.js

```diff
@@ -51,41 +51,52 @@ function (Okta, FormController, FormType) {
              </div>'
         })
       ],
-      _startPolling: function () {
+      _doPoll: function () {
         // start polling request 
         this.transactionObject.poll()
           .then((resp) => {
-            this.options.appState.trigger('setTransaction', resp);
-            const factorPollingInterval = resp.transaction.profile.refresh;
-            let factorPollingIntervalSeconds = Math.floor(factorPollingInterval/1000);
-            // start one second countdown for next poll request
-            this.countDown  = setInterval(() => {
-              // update title after every second and check if countdown == 1 to poll again
-              const title = Okta.loc('polling.title','login', [factorPollingIntervalSeconds]);
-              this.$el.find('.okta-form-title').text(title);
-              this.$('.okta-waiting-spinner').hide();
-              if (factorPollingIntervalSeconds === 0) {
-                /* when countdown hits 0
-                 - stop current poll
-                 - show spinner
-                 - start next poll
-                */
-                this._stopPolling();
-                this.$('.okta-waiting-spinner').show();
-                setTimeout(() => {
-                  this._startPolling();
-                }, 200);
-              } else {
-                // reduce countdown after every second
-                factorPollingIntervalSeconds = factorPollingIntervalSeconds-1;
-              }
-            }, 1000);
+            if (resp.data && resp.status !== 'POLL') {
```

Comment:
Waiting to get confirmation from backend on this check to see if we can only get into SUCCESS state after poll, if so I will update this.

Reply:
going to leave this as is